keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -641,6 +645,16 @@ feature and upgrade/downgrade is not a problem.
 
 ### Version Skew Strategy
 
+When moving from beta to GA, we will introduce a new API group version `apidiscovery.k8s.io/v2`.
+
+All clients v1.26 to v1.29 will only request for the beta API group version `apidiscovery.k8s.io/v1beta2`.
+
+To accomodate skew between the client and server (older client and newer server), the server will serve both v2 and v2beta1 versions based on the client request headers. The apiserver will continue to support v2beta1 until its deprecation in Kubernetes v1.33.
```

Comment:
```suggestion
To accommodate skew between the client and server (older client and newer server), the server will serve both v2 and v2beta1 versions based on the client request headers. The API server will continue to support v2beta1 until its outright removal in Kubernetes v1.33.
```
A release offering (stable) v2 triggers _deprecation_ of all the alpha and beta v2something API versions..

Reply:
Superseded: https://github.com/kubernetes/enhancements/pull/4472/files#r1474732684